velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -898,6 +898,15 @@ TEST_F(DateTimeFunctionsTest, fromUnixtime) {
 
   // 8 hours ahead UTC.
   setQueryTimeZone("Asia/Shanghai");
+// In debug mode, Timestamp constructor will throw exception if range check
```

Comment:
Is there a deeper problem here that Timestamp cannot represent values representable in Spark? If not, then, perhaps there is some issue with the implementation where it attempts to create an "invalid" Timestamp.

Reply:
Presto's Timestamp is stored in one 64-bit signed integer for **milliseconds**, but Spark's Timestamp is stored in 64-bit signed integer for **seconds**, so this test attempts to create an "invalid" Timestamp. Maybe we should remove the range check for Timestamp since it's also used by Spark now.

Reply:
It would be nice to follow-up on that. CC: @rui-mo

Reply:
I will work on it when i have time.

Reply:
@NEUpanning Do we need to create an issue to track this follow-up?

Reply:
Yes, i opened a issue : #9904